content/en/docs/concepts/cluster-administration/cluster-autoscaling.md

```diff
@@ -1,117 +0,0 @@
-
-The [README](https://github.com/kubernetes/autoscaler/tree/c6b754c359a8563050933a590f9a5dece823c836/cluster-autoscaler#readme)
-for the cluster autoscaler lists some of the cloud provider integrations
-that are available.
-
-## Cost-aware multidimensional scaling {#autoscaling-multi-dimension}
-
-### Karpenter {#autoscaler-karpenter}
-
-[Karpenter](https://karpenter.sh/) supports direct node management, via
-plugins that integrate with specific cloud providers, and can manage nodes
-for you whilst optimizing for overall cost.
-
-> Karpenter automatically launches just the right compute resources to
-> handle your cluster's applications. It is designed to let you take
-> full advantage of the cloud with fast and simple compute provisioning
-> for Kubernetes clusters.
-
-The Karpenter tool is designed to integrate with a cloud provider that
-provides API-driven server management, and where the price information for
-available servers is also available via a web API.
-
-For example, if you start some more Pods in your cluster, the Karpenter
-tool might buy a new node that is larger than one of the nodes you are
-already using, and then shut down an existing node once the new node
-is in service.
-
-#### Cloud provider integrations {#karpenter-providers}
-
-{{% thirdparty-content vendor="true" %}}
-
-There are integrations available between Karpenter's core and the following
-cloud providers:
-
-- [Amazon Web Services](https://github.com/aws/karpenter-provider-aws)
-- [Azure](https://github.com/Azure/karpenter-provider-azure)
-
-
-## Related components
-
-### Descheduler
-
-The [descheduler](https://github.com/kubernetes-sigs/descheduler) can help you
-consolidate Pods onto a smaller number of nodes, to help with automatic scale down
-when the cluster has space capacity.
-
-### Sizing a workload based on cluster size
-
-#### Cluster proportional autoscaler
-
-For workloads that need to be scaled based on the size of the cluster (for example
-`cluster-dns` or other system components), you can use the
-[_Cluster Proportional Autoscaler_](https://github.com/kubernetes-sigs/cluster-proportional-autoscaler).<br />
-
-The Cluster Proportional Autoscaler watches the number of schedulable nodes
-and cores, and scales the number of replicas of the target workload accordingly.
-
-#### Cluster proportional vertical autoscaler
-
-If the number of replicas should stay the same, you can scale your workloads vertically according to the cluster size using
-the [_Cluster Proportional Vertical Autoscaler_](https://github.com/kubernetes-sigs/cluster-proportional-vertical-autoscaler).
-This project is in **beta** and can be found on GitHub.
-
-While the Cluster Proportional Autoscaler scales the number of replicas of a workload, the Cluster Proportional Vertical Autoscaler
-adjusts the resource requests for a workload (for example a Deployment or DaemonSet) based on the number of nodes and/or cores
-in the cluster.
```

Comment:
If removing this content from the page about node autoscaling, please move it somewhere else (don't drop it).

Reply:
Good catch!

Reply:
Both of the components mentioned in this section autoscale workloads, not Nodes. Not sure why they were mentioned here in the first place, they're already documented on the workload autoscaling page: https://kubernetes.io/docs/concepts/workloads/autoscaling/#autoscaling-based-on-cluster-size.

Reply:
They're mentioned because a reader who is setting up autoscaling might be interested to know that there is code to size workloads based on cluster size.

(in the scenario where we don't mention them, either readers don't learn that those components exist, or they try a search engine and get lucky)

Reply:
I see your point, but the content on both pages was pretty similar, and seemed a bit redundant. Added a section for node-count-based workload autoscalers in the "Related components" section, linking to the workload autoscaling page to reduce the redundancy. WDYT?

Reply:
Yes, this works.